doc/radosgw/multisite.rst

```diff
@@ -66,10 +66,13 @@ something like this:
 
 At the top of this diagram, we see two applications (also known as "clients").
 The application on the right is both writing and reading data from the Ceph
-Cluster, by means of the RADOS Gateway (RGW).
+Cluster, by means of the RADOS Gateway (RGW). The application on the left is
+only reading data from the Ceph Cluster, by means of an instance of RADOS
```

Comment:
I might italicize _reading_ to disambiguate the message.

Reply:
I'll do that. I appreciate the benefit of your fresh eyes.